website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+
+| ID | Name  | Requirement Statement   | Solution Mapping  |
+| ------ | ---- | --------- | ---------- |
+| DIA-01 | DID Verification | The DIA issuer (registrar) SHOULD confirm that the registered member (subject) is the legitimate holder of a DID before issuing a DIA credential so that the registrar is protected against members falsely claiming ownership of well known DIDs| MAY use the [DID Auth](https://w3c-ccg.github.io/vp-request-spec/#did-authentication) protocol for this purpose. |
+| DIA-02 | DIA Issuer DID | The DIA issuer MUST use did:web (or a trusted variant such as did:tdw) as the DIA issuer and the web domain MUST match the well known domain of the issuing authority so that verifiers can confirm authority identity via public web records.| DIA issuer specification|
+| DIA-03 | Scheme registration | The DIA issuing authority SHOULD register the identity scheme (including the trusted issuer DIDs) with the UN/CEFACT identifier scheme registry so that verifiers can leverage UN maintained scheme metadata to simplify DIA discovery and verification.| See UNTP [Identity Resolver](IdentityResolver.md)|
+| DIA-04 | Subject DID List | The DIA MUST allow multiple subject DIDs to be linked to the same registered identity so long as proof of ownership is verified for each - so that members can rotate multiple active DIDs | `VerfiedDIDList` property|
+| DIA-05 | Scope List| The DIA MUST include a list of scope URIs that unambiguously define the authorised role(s) of the member in the register so that verifiers can confirm the scope of the membership. | `scopeList` property |
+| DIA-06 | Register Type| The DIA MUST specify the register type so that verifiers can understand the context of the `scopeList`| `registerType` property|
+| DIA-07 | DIA Discovery| The DIA SHOULD be discoverable given either the DID or the registeredID| [DIA Discovery](#dia-discovery)|
+
+The examples below help to clarify the application of DIA-05 and DIA-06.
+
+## Logical Model
+
+The Digital Identity Anchor logical model.
+
+![Digital Product Passport data model](DigitalIdentityAnchor.svg)
+
+### Core Vocabulary Documentation
+
+The [UNTP core types vocabulary](https://jargon.sh/user/unece/untp-core/v/0.5.0/artefacts/readme/render) defines the uniquely identified Linked Data entities such as Product, Location, Facility, Party, Standard, Regulation, Criteria, Declaration, Attestation, Endorsement. The Digital Identity Anchor only re-uses the UNTP core `VerifiableCredential` and `Identifier` classes.
+
+
+### DIA Documentation
+
+The [DIA documentation](https://jargon.sh/user/unece/DigitalIdentityAnchor/v/0.1.0/artefacts/readme/render) provides a technology-neutral definition of classes, properties and code lists in the DIA model.
+
+## Implementation Guidance
+
+This section provides sample JSON-LD snippets for each DIA component.
```

Comment:
Well, pseudo LD, actually

Reply:
sounds like a topic for discussion - but hopefully wont block this PR